vendors/ti/boards/cc3220_launchpad/aws_demos/config_files/aws_demo_config.h

```diff
@@ -52,7 +52,7 @@
 #endif
 
 #define democonfigSHADOW_DEMO_NUM_TASKS                ( 1 )
-#define democonfigSHADOW_DEMO_TASK_STACK_SIZE          ( configMINIMAL_STACK_SIZE * 4 )
+#define democonfigSHADOW_DEMO_TASK_STACK_SIZE          ( 1024 )
```

Comment:
To be consistent, can we keep the size in multiples of configMINIMAL_STACK_SIZE  rather than using constants ?

Reply:
We have been changing stack sizes to constants, as configMINIMAL_STACK_SIZE is different for every board.

Reply:
Was this amount chosen because it just a large enough nice power of 2 number?

Reply:
Yes